index.html

```diff
@@ -2,184 +2,185 @@
-      max-height: 100vh;
-      overflow-x: scroll;
-      overflow-y: hidden;
-    }
-
-    .intro {
-      width: 100vw;
-      height: 100vh;
-      object-fit: cover;
-    }
-
-    .scroll {
-      position: absolute;
-      top: 0;
-      left: 100vw;
-      min-width: 100vw;
-      height: 100vh;
-      display: flex;
-    }
-
-    .recap {
-      height: 100%;
-    }
-
-    .recap img {
-      height: 100%;
-      width: auto;
-    }
-
-    .season2 {
-      width: 100vw;
-      height: 100vh;
-      max-height: 100vh;
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      place-content: center;
-      text-align: center;
-    }
-
-    .season2 h1,
-    h2 {
-      line-height: 1em;
-      font-weight: bold;
-    }
-
-    .season2 h1 {
-      font-size: 6em;
-    }
-
-    .season2 h1 span {
-      font-size: 1em;
-      font-weight: bold;
-      color: #cb2529;
-    }
-
-    .season2 h2 {
-      font-size: 3em;
-    }
-
-    .season2 p {
-      margin-top: 1em;
-      font-size: 2em;
-      line-height: 1em;
-    }
-
-    @media (max-width: 800px) {
-      .season2 h1 {
-        font-size: 3em;
-      }
-
-      .season2 h2 {
-        font-size: 2em;
-      }
-
-      .season2 p {
-        font-size: 1em;
-      }
-    }
-  </style>
-
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+    <!-- Primary Meta Tags -->
+    <title>Season 2 | Bankless DAO</title>
+    <meta name="title" content="Season 2 | Bankless DAO" />
+    <meta name="description" content="Recap for Season 1 and intro to Season 2 🚀" />
+    <!-- Open Graph / Facebook (needs to be < 300kb to work on WhatsApp) -->
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://season2.bankless.community/" />
+    <meta property="og:title" content="Season 2 | Bankless DAO" />
+    <meta property="og:description" content="Recap for Season 1 and intro to Season 2 🚀" />
+    <meta property="og:image" content="https://season2.bankless.community/Season1-Meta.jpg" />
+    <meta property="og:image:width" content="600" />
+    <meta property="og:image:height" content="330" />
+    <!-- Twitter -->
+    <meta property="twitter:url" content="https://season2.bankless.community/" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:title" content="Season 2 | Bankless DAO" />
+    <meta name="twitter:description" content="Recap for Season 1 and intro to Season 2 🚀" />
+    <meta name="twitter:image" content="https://season2.bankless.community/Season1-Meta.jpg" />
+    <!-- favicons -->
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="icon" type="image/png" sizes="33x33" href="/favicon.png" />
+    <link rel="apple-touch-icon" type="image/png" sizes="33x33" href="/favicon.png" />
+    <style type="text/css">
+        html,
+        * {
+            margin: 0;
+        }
```

Comment:
let's put a black background on all the page, so we don't see the small white gaps.

Reply:
done!